### PR TITLE
Add mirror debug fluentbit image

### PR DESCRIPTION
**Pull request by nickgerace**
Add mirror debug fluentd image (v1.11.2). Add mirror debug
fluentbit image (1.5.4).

#### Pull Request Checklist ####

- [x] Change does not remove any existing Images or Tags
- [x] New entries are in format `SOURCE DESTINATION TAG`
- [x] New entries are added to the correct section of the list (sorted lexographically) 
- [x] Changes to scripting or CI config have been tested to the best of your ability

#### Types of Change ####

New image

#### Linked Issues ####

[Issue: 28657](https://github.com/rancher/rancher/issues/28657)

#### Additional Notes ####

n/a

### CODE REVIEW

File: images-list [hunk 1]
```diff
@@ -3,12 +3,14 @@
 # https://github.com/rancher/image-mirror/pull/27
 #
 # LIST FORMAT: <SOURCE> <DESTINATION> <TAG>
+banzaicloud/fluentd rancher/banzaicloud-fluentd v1.10.4-alpine-2-debug
```
Comment (paynejacob) [inline images-list:6]: we should mirror the debug image the same as the actual image `v1.11.2-alpine-2-debug`

File: images-list [hunk 2]
```diff
@@ -3,12 +3,14 @@
...
 banzaicloud/fluentd rancher/banzaicloud-fluentd v1.11.2-alpine-2
 banzaicloud/logging-operator rancher/banzaicloud-logging-operator 3.6.0
 bats/bats rancher/bats-bats v1.1.0
 coredns/coredns rancher/coredns-coredns 1.7.0
 curlimages/curl rancher/curlimages-curl 7.70.0
 directxman12/k8s-prometheus-adapter-amd64 rancher/directxman12-k8s-prometheus-adapter-amd64 v0.6.0
+fluent/fluent-bit rancher/fluent-fluent-bit 1.5.0-debug
```
Comment (paynejacob) [inline images-list:13]: `1.5.4-debug`

**Review by paynejacob — CHANGES_REQUESTED** [review]

File: images-list (hunk 1 above)
Comment (nickgerace) [reply images-list]: Good catch. Fixed and I'll note this in the issue filed.

File: images-list (hunk 2 above)
Comment (nickgerace) [reply images-list]: Done.

**Review by paynejacob — DISMISSED** [review]

File: images-list (hunk 1 above)
Comment (cbron) [reply images-list]: This doesn't exist now: https://hub.docker.com/r/banzaicloud/fluentd/tags?page=1&name=v1.11.2-alpine-2

Comment (nickgerace) [reply images-list]: Interesting... I'll touch base with Jacob offline. Doesn't look like there are any ```fluentd``` debug images at the moment.

Comment (nickgerace) [reply images-list]: Confirmed with @paynejacob. The ```debug``` images are [only offered for fluentbit](https://banzaicloud.com/docs/one-eye/logging-operator/troubleshooting/#debug-version-of-the-fluentbit-container), not fluentd. Adjusted the PR accordingly.

**Comment by nickgerace** [comment]
Making a general comment from the review discussion... spoke with @paynejacob: the debug images are only offered for fluentbit, not fluentd.

The PR should now reflect this.

**Review by paynejacob — APPROVED** [review]

**Review by cbron — APPROVED** [review]

**Pull request merged**